### PR TITLE
fix bundler methods

### PR DESCRIPTION
**Pull request by himanshugarg06**
@arcticfloyd1984 can you help with the param definition and other details if any

### CODE REVIEW

File: docs/Bundler/integration.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ custom_edit_url: https://github.com/bcnmy/docs/blob/master/docs/Bundler/integrat
 
 # Integration
 
-This guide soley focuses on the code to integrate the Bundler. To see how this interacts with other packages in the sdk you can view the [Smart Accounts Integration page](/Account/integration).
+This guide solely focuses on the code to integrate the Bundler. To see how this interacts with other packages in the sdk you can view the [Smart Accounts Integration page](/Account/integration).
```
Comment (NunoFilipeSantos) [inline docs/Bundler/integration.mdx:8]:
```suggestion
This guide solely focuses on the code to integrate the Bundler. To see how this interacts with other packages in the SDK you can view the [Smart Accounts Integration page](/Account/integration).
```

**Review by NunoFilipeSantos — COMMENTED** [review]
Please capitalize acronyms like SDK, URL.

File: docs/Bundler/integration.mdx [hunk 2]
```diff
@@ -5,7 +5,7 @@ custom_edit_url: https://github.com/bcnmy/docs/blob/master/docs/Bundler/integrat
 
 # Integration
 
-This guide soley focuses on the code to integrate the Bundler. To see how this interacts with other packages in the sdk you can view the [Smart Accounts Integration page](/Account/integration).
+This guide focuses on the code to integrate the Bundler. To see how this interacts with other packages in the sdk you can view the [Smart Accounts Integration page](/Account/integration).
```
Comment (NunoFilipeSantos) [inline docs/Bundler/integration.mdx:8]:
```suggestion
This guide focuses on the code to integrate the Bundler. To see how this interacts with other packages in the SDK you can view the [Smart Accounts Integration page](/Account/integration).
```

File: docs/Bundler/integration.mdx [hunk 3]
```diff
@@ -46,51 +46,36 @@ pnpm add @biconomy/bundler @biconomy/account
...
+- [userOpWaitForTxHashIntervals](https://bcnmy.github.io/biconomy-client-sdk/classes/Bundler.html#UserOpWaitForTxHashIntervals)(`object`, optional): The polling interval per chain for the tx result in milliseconds. Default value is 0.5 seconds.
+- [userOpReceiptMaxDurationIntervals](https://bcnmy.github.io/biconomy-client-sdk/classes/Bundler.html#userOpReceiptMaxDurationIntervals)(`object`, optional): The maximum duration in milliseconds per chain to wait for the tx receipt. Default value is 30 seconds.
+- [userOpWaitForTxHashMaxDurationIntervals](https://bcnmy.github.io/biconomy-client-sdk/classes/Bundler.html#UserOpWaitForTxHashMaxDurationIntervals)(`object`, optional): The maximum duration in milliseconds per chain to wait for the tx hash. Default value is 20 seconds.
+
+**returns**
+- [bundler](https://bcnmy.github.io/biconomy-client-sdk/classes/Bundler.html)(`Promise<Bundler>`): bundler instance
+
+Once initialized you will be able to pass this Bundler instance to the Smart Account configuration to use in conjunction with Smart Accounts. See our [tutorials](/tutorials) for in depth integrations of the Smart Account and Bundler.
```
Comment (NunoFilipeSantos) [inline docs/Bundler/integration.mdx:81]:
```suggestion
- bundlerUrl(`string`, required): bundler URL from the dashboard
- [userOpReceiptIntervals](https://bcnmy.github.io/biconomy-client-sdk/classes/Bundler.html#UserOpReceiptIntervals)(`object`, optional): The polling interval per chain for the tx receipt in milliseconds. The default value is 5 seconds.
- [userOpWaitForTxHashIntervals](https://bcnmy.github.io/biconomy-client-sdk/classes/Bundler.html#UserOpWaitForTxHashIntervals)(`object`, optional): The polling interval per chain for the tx result in milliseconds. The default value is 0.5 seconds.
- [userOpReceiptMaxDurationIntervals](https://bcnmy.github.io/biconomy-client-sdk/classes/Bundler.html#userOpReceiptMaxDurationIntervals)(`object`, optional): The maximum duration in milliseconds per chain to wait for the tx receipt. The default value is 30 seconds.
- [userOpWaitForTxHashMaxDurationIntervals](https://bcnmy.github.io/biconomy-client-sdk/classes/Bundler.html#UserOpWaitForTxHashMaxDurationIntervals)(`object`, optional): The maximum duration in milliseconds per chain to wait for the tx hash. The default value is 20 seconds.

**returns**
- [bundler](https://bcnmy.github.io/biconomy-client-sdk/classes/Bundler.html)(`Promise<Bundler>`): bundler instance

Once initialized you will be able to pass this Bundler instance to the Smart Account configuration to use in conjunction with Smart Accounts. See our [tutorials](/tutorials) for in-depth integrations of the Smart Account and Bundler.
```

**Review by NunoFilipeSantos — CHANGES_REQUESTED** [review]
Typos.

**Review by arcticfloyd1984 — APPROVED** [review]

File: docs/Bundler/bundlermethods.md [hunk 4]
```diff
@@ -4,86 +4,134 @@ sidebar_position: 3
 custom_edit_url: https://github.com/bcnmy/docs/blob/master/docs/Bundler/integration.mdx
 ---
 
-# Bundler Methods
+# Methods
 
-Following are the methods that can be called on bundler instance
+## [estimateUserOpGas](https://bcnmy.github.io/biconomy-client-sdk/classes/Bundler.html#estimateUserOpGas)
+This method is used to estimate gas for the userOp. It returns estimates for PreVerificationGas, VerificationGas, and CallGasLimit for a given UserOperation. It requires passing a semi-valid/ dummy signature in userOp (e.g. a signature of the correct length and format).
```
Comment (arcticfloyd1984) [inline docs/Bundler/bundlermethods.md:10]: Should be verificationGasLimit not VerificationGas
Can we also make it use camel casing for these variables?

**Review by arcticfloyd1984 — APPROVED** [review]

**Pull request merged**